centos.org/pipelines/lib/foremanDuffyJob.groovy

```diff
@@ -66,22 +81,23 @@ pipeline {
                     """
 
                     writeFile(file: 'otel_env', text: otel_env)
-                    duffy_scp_in('otel_env', 'otel_env', 'duffy_box', './')
-                    sh(script: 'rm -rf otel_env', label: 'remove otel_env file')
 
                     def branches = [:]
                     pipelines.each { action, oses ->
                         oses.each { os ->
                             def name = "${os}-${action}"
+                            def username = "pipe-${name}"
+                            def boxname = "${username}@duffy_box"
                             branches[name] = {
                                 def playBook = pipelineVars(action: action, type: params.type, version: params.version, os: os, extra_vars: ['expected_version': params.expected_version ?: ''])
-                                def extra_vars = buildExtraVars(extraVars: playBook['extraVars'] + ["forklift_vagrant_lock": "/tmp/forklift.lock"])
-                                def playbooks = duffy_ssh("ls forklift/pipelines/${playBook['pipeline']}", 'duffy_box', './', true)
+                                def extra_vars = buildExtraVars(extraVars: playBook['extraVars'])
+                                def playbooks = duffy_ssh("ls forklift/pipelines/${playBook['pipeline']}", boxname, './', true)
                                 playbooks = playbooks.split("\n")
 
+                                duffy_scp_in('otel_env', 'otel_env', boxname, './')
```

Comment:
Not for this PR, but I was looking at this change myself and wondered about using [sshGet](https://www.jenkins.io/doc/pipeline/steps/ssh-steps/#sshget-ssh-steps-sshget-get-a-filedirectory-from-remote-node), [sshPut](https://www.jenkins.io/doc/pipeline/steps/ssh-steps/#sshput-ssh-steps-sshput-put-a-filedirectory-on-remote-node) and [sshCommand](https://www.jenkins.io/doc/pipeline/steps/ssh-steps/#sshcommand-ssh-steps-sshcommand-execute-command-on-remote-node) instead. Have you thought about that?

Reply:
I did not know about that, so, no. But certainly something to look at.